client/src/components/BirthDate/BirthDate.tsx

```diff
@@ -0,0 +1,77 @@
+import { useState } from 'react';
+import { FormControl, Grid, MenuItem, Select, SelectChangeEvent } from '@mui/material';
+
+const BirthDate = (): JSX.Element => {
+  const [year, setYear] = useState<string>('1998');
+  const [month, setMonth] = useState<string>('January');
+  const [day, setDays] = useState<string>('12');
+  const handleYearChange = (event: SelectChangeEvent) => {
+    setYear(event.target.value as string);
+  };
+  const handleMonthChange = (event: SelectChangeEvent) => {
+    setMonth(event.target.value as string);
+  };
+  const handleDayChange = (event: SelectChangeEvent) => {
+    setDays(event.target.value as string);
+  };
+  const getYear = new Date().getFullYear();
+  // To Get Years
+  const getYears = Array.from(new Array(100), (val, index) => getYear - index);
+  const getMonth = [
+    'January',
+    'February',
+    'March',
+    'April',
+    'May',
+    'June',
+    'July',
+    'August',
+    'September',
+    'October',
+    'November',
+    'December',
+  ];
+
+  const getDays = Array.from(new Array(31), (val, index) => index + 1);
```

Comment:
Probably better to leverage current date libraries to make this work. I'm thinking we'll likely just swap this field for just a simple date field instead though.